### ISSUE TITLE
what are the major differences between fintuning QwenImage and QwenImageLayer

### ISSUE THREAD

**Issue by garychan22** [modelscope/DiffSynth-Studio#1274]
Hi, I am now using my own scripts implemented with diffusers and accelerate to finetune QwenImageLayer and I found that the convergence is slow and even it didn't converge with large loss around 0.84.

My modification from QwenImage:
1. RGBA-VAE encoding
2. 3Drope is constructed in the order of [[recontructed_image, layer00, layer01, ..., cond_image]]
3. resolution 640

I found a released training script here https://github.com/modelscope/DiffSynth-Studio/blob/main/examples/qwen_image/model_training/full/Qwen-Image-Layered.sh. However, I am new to DiffSynth-Studio.

Can someone share the major differences between finetuning QwenImage and QwenImageLayer, thanks!

**Comment by lqy9715** [comment]
hello,I have also been training the qwen image layer model with Lora and full parameters recently. Can you add a WeChat contact information for easy communication.

**Comment by garychan22** [comment]
> hello,I have also been training the qwen image layer model with Lora and full parameters recently. Can you add a WeChat contact information for easy communication.

sure, ``WinneChan666`` @lqy9715

**Comment by garychan22** [comment]
possibly already fixed the training issue with diffusers scripts
